modules/accumulo/src/main/java/io/fluo/recipes/accumulo/ops/TableOperations.java

```diff
@@ -118,7 +123,10 @@ public static void compactTransient(FluoConfiguration fluoConfig) throws Excepti
       TransientRegistry transientRegistry = new TransientRegistry(appConfig);
       List<RowRange> ranges = transientRegistry.getTransientRanges();
 
+      Logger logger = LoggerFactory.getLogger(TableOperations.class);
```

Comment:
I was thinking it was only used in that method.   However I just double checked and another method is logging, so I will make it static.